lib/resty/mlcache/ipc.lua

```diff
@@ -137,14 +149,18 @@ function _M:broadcast(channel, data)
 end
 
 
-function _M:poll()
+function _M:poll(max_sleep)
```

Comment:
I can see that, but if for my usecase I need 0.3 seconds, I now have to make sure that all my calls ever will pass that timeout. That's cumbersome.

Question here is whether there is a usecase for per poll timeouts.